stream-applications-release-train/stream-applications-docs/src/main/asciidoc/index.adoc

```diff
@@ -15,13 +15,15 @@ Sabby Anandan; Artem Bilan; Marius Bogoevici; Eric Bottard; Mark Fisher; Ilayape
 
 :stream-apps-asciidoc: https://raw.githubusercontent.com/spring-cloud/stream-applications/main/stream-applications-release-train/stream-applications-docs/src/main/asciidoc
 
-:scst-core-version: 3.1.5.RELEASE
+:spring-core-version: 5.3.20
+
+:scst-core-version: 3.2.0
 
 ifdef::backend-html5[]
 
 Version {project-version}
 
-(C) 2012-2021 Pivotal Software, Inc.
+(C) 2012-2022 Pivotal Software, Inc.
```

Comment:
I guess this one has to be now like `VMware, Inc. All rights reserved.`

Reply:
Yep, it crossed my mind as well. It is displayed on [index page at top](https://docs.spring.io/stream-applications/docs/current/reference/html/). 

Other ref docs looks like they have the copyright date but have no company name (ex: [Spring-Boot](https://docs.spring.io/spring-boot/docs/current/reference/html/legal.html#legal)).

I will remove the name portion out of this.

Reply:
Done